vhdbuilder/packer/components.json

```diff
@@ -41,6 +41,7 @@
       "amd64OnlyVersions": [],
       "multiArchVersions": [
         "v1.4.44.5",
```

Comment:
Can we remove the old one if it is superseded?

Reply:
@juan-lee the old one is still present in production, we can remove it after this new version rolls out

Reply:
should be safe to remove since this is just what we cache on the VHD during build-time _moving forward,_ what's currently in production won't be affected

Reply:
but you could migrate to this VHD before getting the updated addon version, no? in that case all your nodes are going to need to pull the image from MCR at boot